javascript/javascript2/week1/index.js

```diff
@@ -0,0 +1,64 @@
+//Find the shortest word
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+getShortestWord(danishWords);
+
+function getShortestWord(words) {
```

Comment:
Good name. But when naming a function get.... one would expect to get a return value. This function only logs. So perhaps logShortestWord would be more appropriate

Reply:
thanks for the feedback. changed the function name